Dockerfile

```diff
@@ -11,25 +11,27 @@ RUN test -n "${IBC_GO_VERSION}"
 COPY go.mod .
 COPY go.sum .
 
-RUN go mod download
-
+ADD modules modules
 ADD internal internal
 ADD testing testing
-ADD modules modules
 ADD LICENSE LICENSE
 
+RUN go mod download
+
 COPY contrib/devtools/Makefile contrib/devtools/Makefile
 COPY Makefile .
 
-
 RUN make build
+# Grab the wasmvm library in order to copy it into the final image.
+RUN cp $(ldd build/simd | grep "libwasmvm" | awk '{ print $3 }') libwasmvm.so
```

Comment:
A question. Do we want to keep using multi-stage builds? This (and the copy in the next image) is required because `libwasmvm` is included in the `wasmvm` go lib (which we lose access to in the next stage).

Are the space savings worth the hassle?

Reply:
Note, they are worth it. Looking into https://github.com/cosmos/ibc-go/issues/2442 though since that might offer benefits in reducing issues like this.